packages/@react-native-windows/cli/src/runWindows/runWindows.ts

```diff
@@ -121,7 +121,6 @@ async function runWindows(
       name: 'run-windows',
       properties: {
         release: options.release,
-        root: options.root !== undefined,
```

Comment:
root is always defined since there’s a fallback if there is no CLI arg. No use sending “true” for every session.